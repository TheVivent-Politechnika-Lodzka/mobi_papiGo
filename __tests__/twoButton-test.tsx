import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import TwoButton from '../src/components/twoButton';


it('Button test', () => {
  const valueTwo = 'test';
  const card = renderer.create(<TwoButton valueOne='5' valueTwo={valueTwo} onPressOne={() => {}} onPressTwo={() => {}}/>).toJSON();
  expect(card).toMatchSnapshot();
});