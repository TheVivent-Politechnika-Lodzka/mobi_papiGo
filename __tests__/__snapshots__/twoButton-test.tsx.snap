// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test 1`] = `
<View>
  <View
    style={
      Array [
        Object {
          "borderRadius": 3,
          "overflow": "hidden",
        },
        Object {
          "borderRadius": 3,
        },
        Object {
          "marginHorizontal": 50,
          "marginVertical": 10,
          "width": 200,
        },
        false,
      ]
    }
  >
    <View
      accessibilityRole="button"
      accessibilityState={
        Object {
          "busy": false,
          "disabled": false,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#C8E3D4",
            "borderColor": "white",
            "borderRadius": 30,
            "borderWidth": 1,
            "flexDirection": "row",
            "justifyContent": "center",
            "padding": 8,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#000",
              "fontSize": 18,
              "fontWeight": "bold",
              "paddingVertical": 1,
              "textAlign": "center",
            }
          }
        >
          5
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "borderRadius": 3,
          "overflow": "hidden",
        },
        Object {
          "borderRadius": 3,
        },
        Object {
          "marginHorizontal": 50,
          "marginVertical": 10,
          "width": 200,
        },
        false,
      ]
    }
  >
    <View
      accessibilityRole="button"
      accessibilityState={
        Object {
          "busy": false,
          "disabled": false,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#C8E3D4",
            "borderColor": "white",
            "borderRadius": 30,
            "borderWidth": 1,
            "flexDirection": "row",
            "justifyContent": "center",
            "marginBottom": "10%",
            "padding": 8,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#000",
              "fontSize": 18,
              "fontWeight": "bold",
              "paddingVertical": 1,
              "textAlign": "center",
            }
          }
        >
          test
        </Text>
      </View>
    </View>
  </View>
</View>
`;
